contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,29 +6,63 @@ showTitle: true
 
 ## Our mission
 
-Equip every developer to build successful products
+Equip every developer to build successful products.
```

Comment:
Is this still the mission? 🙈 We don't talk about it much, which kind of makes me think it isn't resonating as much as it once did. Maybe we need to either rethink it, or make a purpose out of talking about it more.